### ISSUE TITLE
i can't reach gophish on my localhost3333

### ISSUE THREAD

**Issue by playjett** [gophish/gophish#1030]
please aid me out,i cannot reach my gophish on my localhost, i also install git bash to run the comans but i don't seem to get my command right to it. When i cd into it the gophish shows in yellowish and the master shows in blue like this  MINGW64 ~/gophish (master)
am using windows 8.1 but i reach my host ip address 120.0.0.1 opens.

**Comment by playjett** [comment]
am using windows 8.1

**Comment by S0larflare** [comment]
Sorry, it is a bit hard to follow what you actually mean here. Could you follow the template and provide a screenshot of what you mean?

Also, did you mean 127.0.0.1?

**Comment by jordan-wright** [comment]
Hi @playjett, thanks for reaching out. In order to help troubleshoot we're going to require quite a bit more information. This information is requested by the issue template you saw when you first created a new issue.

Since this doesn't follow the issue template, I'm going to go ahead and close this out. If you could, please create a new issue, following the issue template as closely as possible. That will give us the information we need to help point you in the right direction.

Thanks again for getting in touch!

**Comment by playjett** [comment]
THIS IS WHAT MY GOPHISH DOES ANYTIME I TRY TO USE IT, I HAVE TRIED ALMOST ALL THE SMTP SERVERS I KNOW BUT IS STILL THE SAME PLEASE HELP.this is what it looks like.
![gophish file](https://user-images.githubusercontent.com/37704143/48597328-21c45c00-e912-11e8-84cb-c1c22221e1e5.PNG)
